lib/iex/test/iex/helpers_test.exs

```diff
@@ -339,7 +339,7 @@ defmodule IEx.HelpersTest do
       assert capture_io(fn -> h(:timer.sleep() / 1) end) == """
              * :timer.sleep/1
 
-               @spec sleep(time) :: :ok when Time: timeout(), time: var
+               @spec sleep(time) :: :ok when Time: timeout()
```

Comment:
I still don't think this is correct. The function refers to a variable `time` while the guard specifies `Time`. I think they should be the same in both places.

Reply:
so you think it should be `@spec sleep(time) :: :ok when time: timeout()` ?

Reply:
Yes, I think so. If we're going for elixir syntax everywhere, let's go all the way.

On the other hand, I think printing everything in erlang syntax could be a reasonable choice too, but I think it would be slightly harder and less useful for newcomers.

Reply:
👍 good call. I agree. Will push the change in a few mintues.